doc/classes/ProjectSettings.xml

```diff
@@ -1187,6 +1187,9 @@
 		<member name="rendering/lossless_compression/webp_compression_level" type="int" setter="" getter="" default="2">
 			The default compression level for lossless WebP. Higher levels result in smaller files at the cost of compression speed. Decompression speed is mostly unaffected by the compression level. Supported values are 0 to 9. Note that compression levels above 6 are very slow and offer very little savings.
 		</member>
+		<member name="rendering/mesh_storage/split_stream" type="bool" setter="" getter="" default="false">
+			On import, mesh vertex data will be split into two streams within a single vertex buffer, one for position data and the other for interleaved attributes data. Recommended to be enable if targeting mobile devices. Requires manual reimport of meshes after toggling
```

Comment:
```suggestion
			On import, mesh vertex data will be split into two streams within a single vertex buffer, one for position data and the other for interleaved attributes data. Recommended to be enabled if targeting mobile devices. Requires manual reimport of meshes after toggling.
```

Reply:
Updated the word here!

Reply:
You missed it in my diff but there's also a dot that should be added at the end of the sentence.

Reply:
Oh shoot you're so right sorry about that I hope that takes care of it now!

Reply:
I think you might have missed pushing that last amend?

Reply:
Ah you're right I noticed in the logs that I was getting unresolved host errors in my repo for some reason